scripts/bad_words.py

```diff
@@ -0,0 +1,67 @@
+"""
+
+Usage:
+    scripts/bad_words.py <data_api_url> <data_api_token> <bad_words_path> <framework_slug>
+"""
+from docopt import docopt
+from dmutils.apiclient import DataAPIClient
+import six
+import re
+
+
+def main(data_api_url, data_api_token, bad_words_path, framework_slug):
+    client = DataAPIClient(data_api_url, data_api_token)
+    bad_words = get_bad_words(bad_words_path)
+    suppliers = get_suppliers(client, framework_slug)
+    get_services_with_bad_words(client, suppliers, bad_words)
+
+
+def get_suppliers(client, framework_slug):
+    suppliers = client.find_framework_suppliers(framework_slug)
+    suppliers = suppliers["supplierFrameworks"]
+    if (framework_slug == "g-cloud-6"):
+        suppliers_on_framework = suppliers
+    else:
+        suppliers_on_framework = [supplier for supplier in suppliers if supplier["onFramework"]]
+    return suppliers_on_framework
+
+
+def get_draft_services(client, supplier_id, framework_slug):
+    services = client.find_draft_services(supplier_id, framework=framework_slug)
+    services = services["services"]
+    submitted_services = [service for service in services if service["status"] == "submitted"]
+    return submitted_services
+
+
+def get_services(client, supplier_id, framework_slug):
+    services = client.find_services(supplier_id, framework=framework_slug)
+    services = services["services"]
+    return services
+
```

Comment:
`get_services` and `get_draft_services` could be one method that calls different methods on the API client depending on the framework slug.